contrib/dnsupdate

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# This queries the bitcoin-seeder via dns on port 5353 and 
+# puts the result in the dns zone of a Bind9.
+# bitcoin-seeder: https://github.com/sipa/bitcoin-seeder
+
```

Comment:
would be nicer with  `set -euxo pipefail`

Reply:
I have added `set -euo pipefail`

It should print nothing so no `-x` added.